go-controller/pkg/node/gateway_localnet.go

```diff
@@ -3,7 +3,9 @@
 package node
 
 import (
+	//"debug/elf"
 	"fmt"
+	"github.com/vishvananda/netlink"
```

Comment:
This should be below with the other github imports.

Well... the `"k8s.io/klog"` import should be moved to be with the other k8s.io imports, and then the netlink import would go at the end of the github imports where the klog import currently is...

Reply:
Will do. I thought go formatting will fix all these which I run before committing.